openedx/core/djangoapps/cors_csrf/decorators.py

```diff
@@ -22,6 +23,24 @@ def _inner(*args, **kwargs):  # pylint: disable=missing-docstring
             # the cross-domain version of the CSRF cookie.
             request = args[0]
             request.META['CROSS_DOMAIN_CSRF_COOKIE_USED'] = True
+            current_authenticator = getattr(request, 'successful_authenticator', None)
+
+            if hasattr(request, "resolver_match"):
```

Comment:
Could all 17 of these be added at once? Maybe that isn't an issue at all, but maybe we'd want to put this all behind a `CORS_CSRF_DETAIL_MONITORING` toggle?

That said, I have no idea what data you will actually need, so it seems good that you are covering so much. I wonder if you could put together a plan of what you'd want to review and compare once we have this data?